src/clr-angular/forms/styles/_input-group.clarity.scss

```diff
@@ -6,8 +6,7 @@
   .clr-input-group {
```

Comment:
Was the PR marked as a breaking change due to the visual changes needed to get the alignments correct or was it a API change needing a new CSS class?

Reply:
I don't see this as a breaking change. The change here is fixing a problem.